### PR TITLE
updated landing page and footer

### PR DESCRIPTION
**Pull request by KenMain5**
Mainly CSS changes, Centering search button, removing carousel,

### CODE REVIEW

File: src/lib/Footer.js [hunk 1]
```diff
@@ -1,44 +1,42 @@
...
-        'name': 'Justin Cordova', 
-        'github': 'xyz', 
-        'linkedin': 'xyz'
-    }, 
-    {
-        'name': 'Steven Kim', 
-        'github': 'xyz', 
-        'linkedin': 'xyz'
-    }, 
+    [   'Bruce Wong',
```
Comment (jose-m-f) [inline src/lib/Footer.js:2]: Thanks for putting up our info

**Review by stevkim — APPROVED** [review]
Went through together. -Minor changes to the information but otherwise good

File: src/components/footer/Footer.jsx [hunk 2]
```diff
@@ -5,10 +5,9 @@ function Footer() {
   return (
     <footer className='bg-white-s pb-10'>
         <div className='flex flex-col items-center py-5'>
-          <span className="text-xl font-bold">ParKING</span>
-          <span className="text-xl font-bold">Developers</span>
+          <span className="text-xl font-bold">ParKing Developers</span>
```
Comment (jose-m-f) [inline src/components/footer/Footer.jsx:8]: This looks better than 2 lines

**Review by jose-m-f — APPROVED** [review]
Looks good!

**Pull request merged**